templates/parts/loop/page-content.php

```diff
@@ -10,6 +10,15 @@
 
 	<?php
 
+	/**
+	 * primer_before_content hook.
+	 *
+	 * @hooked primer_woo_shop_message - 10
+	 *
+	 * @since 1.0.0
+	 */
+	do_action( 'primer_before_content' );
```

Comment:
@fjarrett  I had considered this, but went with re-using the hook inside the page template as well. I did this to avoid complexity in hook names, but looking back on it I think you're right. 

To maintain the same naming convention, do you think we should change primer_before_content to `primer_before_single_content`/`primer_before_page_content`?